config/software/autoconf.rb

```diff
@@ -16,7 +16,7 @@
 # expeditor/ignore: deprecated 2021-04
 
 name "autoconf"
-default_version "2.69"
+default_version "2.71"
```

Comment:
flipping the default here may cause some issues downstream.

i'm pretty sure we had to roll back the homebrew version of autoconf to 2.69 for something in CI.  if client/server/workstation all build though it might be fine, but this is probably a good one to validate

Reply:
Please Ignore this PR, as the software has deprecated.